scripts/autoreduction/reduce_EQSANS.py

```
@@ -82,7 +162,7 @@ def upload_plot(run_number: str, plot_div: str):
     try:
         publish_plot("EQSANS", run_number, files={"file": plot_div})
     except requests.HTTPError as e:
-        logging.exception(f"Publish plot failed with error {e}")
+        logging.getLogger(LOG_NAME).error(f"Publish plot failed with error {e}")
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)